frontend/components/draft/metadata/request-review.tsx

```
@@ -1,11 +1,11 @@
 "use client";
-
 import { Button } from "@/components/ui/button";
 import { updateDroplet } from "@/lib/requests/droplet";
 import { Droplet } from "@/types";
 import { CircleHelp } from "lucide-react";
```

ESLint

[error] 'CircleHelp' is defined but never used.

(@typescript-eslint/no-unused-vars)